src/afl-fuzz.c

```diff
@@ -1555,13 +1555,6 @@ int main(int argc, char **argv_orig, char **envp) {
 
   #endif
 
-  // silently disable deterministic mutation if custom mutators are used
```

Comment:
why are you removing this?

Reply:
the same logic is also at line 2193.

Reply:
ok thanks. please write something like that next time you send a PR, I do not have 100% of the code in my head, plus multiple people have written it - this saves the back and forth :)